mongo/options/clientbulkwriteoptions.go

```diff
@@ -0,0 +1,136 @@
+// Copyright (C) MongoDB, Inc. 2024-present.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License. You may obtain
+// a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+package options
+
+import (
+	"go.mongodb.org/mongo-driver/v2/mongo/writeconcern"
+)
+
+// ClientBulkWriteOptions represents options that can be used to configure a client-level BulkWrite operation.
+type ClientBulkWriteOptions struct {
+	// If true, writes executed as part of the operation will opt out of document-level validation on the server. The
+	// default value is false. See https://www.mongodb.com/docs/manual/core/schema-validation/ for more information
+	// about document validation.
+	BypassDocumentValidation *bool
```

Comment:
We no longer add documentation to the options structs as of [GODRIVER-3327](https://jira.mongodb.org/browse/GODRIVER-3327), opting to defer to the setter functions.